syllabus/5-Substrate/3-Merklized-Storage_Slides.md

```diff
@@ -637,60 +619,35 @@ Notes:
 
 ### Overlay
 
-- There is a limit to how many nested layers you can spawn
-- It is not free, thus it is attack-able.
-
-```rust
-with_storage_layer(|| {
-    let foo = sp_io::storage::read(b"foo");
-    with_storage_layer(|| {
-        sp_io::storage::set(b"foo", b"foo");
-        with_storage_layer(|| {
-            sp_io::storage::set(b"bar", foo);
-            with_storage_layer(|| {
-                sp_io::storage::set(b"foo", "damn");
-                Err("damn")
-            })
-            Ok("what")
-        })
-        Err("the")
-    });
-    Ok("hell")
-})
-```
-
----v
-
-### Overlay
-
 - What if I call `sp_io::storage::root()` in the middle of the block?
 - Can the overlay respond to this?
 
 Notes:
 
-NO! overlay works on the level on key-values, ot knows nothing of trie nodes, and to compute the
-root we have to go to the trie layer and pull a whole lot of data back from the disk and build all
-the nodes etc. etc.
+NO!
+The overlay works on the level on key-values, it knows nothing of trie nodes, and to compute the root we have to go to
+the trie layer and pull a whole lot of data back from the disk and build all the nodes etc.
 
 ---v
 
 ### Overlay: More Caches
 
-- There are more caches in the trie layer as well. But outside of the scope of this lecture.
+- There are more caches in the trie layer as well.
+  But outside of the scope of this lecture.
 
 ```bash
 ./substrate --help | grep cache
 ```
 
 Notes:
 
-https://www.youtube.com/embed/OoMPlJKUULY
+<https://www.youtube.com/embed/OoMPlJKUULY>
```

Comment:
I'm not sure why you altered the links here. These are inside the notes and would show correctly. Adding the <> makes them actually wrong

```suggestion
https://www.youtube.com/embed/OoMPlJKUULY
```